### ISSUE TITLE
If the ltr is the trapezoidal schedule?

### ISSUE THREAD

**Issue by solon37** [SirRob1997/Crowded-Valley---Results#4]
![image](https://user-images.githubusercontent.com/60464034/124537766-1caed500-de4d-11eb-84d7-f021511d231b.png)

**Comment by solon37** [comment]
how to ues these schedule,i can not find the manual.
I look forward to  your reply.

**Comment by fsschneider** [comment]
Yes, the `ltr` refers indeed to the (**l**ong) **tr**apezoidal schedule.

In Figure 8 (in the current arXiv version: https://arxiv.org/pdf/2007.01547.pdf) you can find an illustration of the different schedules. Here, the red line for the trapezoidal schedule describes the `ltr`.

![schedules_plot](https://user-images.githubusercontent.com/12153723/124572194-bfb12000-de48-11eb-8625-65e9f0e741f5.png)

The warm-up and cool-down periods are always 1/10 of the total number of epochs.

**Comment by solon37** [comment]
How to use these schedules in pytorch version?

**Comment by fsschneider** [comment]
As described in the[ DeepOBS documentation](https://deepobs.readthedocs.io/en/v1.2.0-beta0_a/api/pytorch/runner/lrrunner.html) the `LearningRateScheduleRunner` for PyTorch can take a list of so-called `lr_sched_epochs` and `lr_sched_factors`. The first describes the epochs where the learning rate should be adjusted, the second list describes the corresponding factory by which to adjust the learning rate.

Building a schedule such as the `ltr` shown above is then just a matter of building those two lists based on the total number of epochs of the problem.
